agenta-backend/agenta_backend/services/evaluation_service.py

```diff
@@ -681,7 +682,9 @@ async def create_new_evaluation(
         organization=app.organization,
         user=app.user,
         testset=testset,
-        status=EvaluationStatusEnum.EVALUATION_STARTED,
+        status=Result(
+            value=EvaluationStatusEnum.EVALUATION_STARTED, type="status", error=None
```

Comment:
using type=status is not nice to be honest. I think just creating another object EvalStatus would be better. Being explicit is always better imho